src/chapters/jdbc-auth.xml

```diff
@@ -299,6 +321,60 @@ Type "help" for help.
                     to run any database upgrade scripts.</para>
             </important>
         </section>
+        <section xml:id="jdbc-auth-sqlserver">
+            <title>SQL Server</title>
+            <para>If using SQL Server, the database and schema must be created first.  The example below assumes
+                that you are running SQL Server on Linux, using the command-line tools to manage it.</para>
```

Comment:
I suggest also mentioning that any tool which can execute SQL against the SQL Server database should suffice.

Reply:
Fixed.